### DISCUSSION TITLE
Fake player for volume control

### DISCUSSION THREAD

**Discussion by zarss** [sle118/squeezelite-esp32#397] [Q&A] [answered]
Originally I was planning to use bare esp32 and rotary encoder for adjusting the volume on a set of synchronized players. Si I ordered some, but starting to think that it will not work if there's no Dac attached to it. What are your thoughts about this?

**Comment by Renber78** [comment]
To be confirmed but this option (Change drive) does not seem to be available on the ESP32 version. On the other hand, no worries for use without a DAC.

**Comment by zarss** [comment]
Any suggestions on settings? I have two speakers who are working fine. Then I turn on ESP32-WROVER development board called "vol" with one LED and rotary encoder connected. All three devices are synchronized including volume. Once "vol" is connected, speakers keep dropping radio playback almost once a minute. When "vol" is unplugged, speakers play radio fine all day. Something must be wrong in the "vol" settings, any suggestions please?

**Reply by zarss** [reply]
This is the only error message I have managed to see: "W (392382) httpd_txrx: httpd_sock_err: error in recv : 104 "  
Device is less than 2m away from a router.

**Reply by Renber78** [reply, marked as answer]
If your goal is to have a volume button in another room, I don't think SqueezeliteESP32 is a good solution for you.
You should create a small Arduino code that would directly send the requests to the LMS server. 
Much cleaner... 
If you don't know how to code, there is probably a similar project that already exists otherwise chatGPT can do that.

**Reply by Renber78** [reply]
searching a little I found this: 
https://github.com/jajito/LMS-ESP32-Monitor-ZX2D10GE01R-V4848

**Reply by zarss** [reply]
Thank you for a prompt and valuable reply. Like leading blind on the right path! I made my volume control with ESPHome now. Thank you!
